vm/chef/cookbooks/singlefs/files/report_stats.py

```diff
@@ -0,0 +1,143 @@
+#!/usr/bin/python
+"""Additional monitoring for the single node filer."""
+
+# Copyright 2019 Google Inc.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+# TODO(leffler): Integrate into collectd configuration.
+
+import socket
+import subprocess
+import time
+
+import singlefs_util as util
+
+hostname = socket.gethostname()
+stats = []
+
+# Performance numbers on 20 Nov 2015. See this link for details:
+# https://cloud.google.com/compute/docs/disks/#pdchart
```

Comment:
Please refresh the max limits in the functions below - the numbers have changed.